tsconfig.json

```
@@ -15,7 +15,7 @@
     "module": "es2015",
     "esModuleInterop": true,
     "outDir": "./dist",
-    "lib": ["es2015", "esnext.asynciterable", "dom"],
+    "lib": ["es2015", "esnext.asynciterable", "dom", "es2020.promise"],
```

We should be okay to remove `es2020.promise`, thanks to https://github.com/apollographql/apollo-client/pull/6504/commits/1dab28f2e2ff42c71492ab886edd03e269cf4fbf.

```suggestion
    "lib": ["es2015", "esnext.asynciterable", "dom"],
```